app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { connect } from 'react-redux';
+import { createStructuredSelector } from "reselect";
+
+const GET_MESSAGES_REQUEST = 'GET_MESSAGES_REQUEST';
+const GET_MESSAGES_SUCCESS = 'GET_MESSAGES_SUCCESS';
+
+function getMessages() {
+  console.log("Action!");
```

Comment:
- [ ] <b>REQUIRED:</b> Kindly keep your codebase clean by making sure to remove all debugging code like `console.log` when you no longer need them.